src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingSynchronisationService.kt

```diff
@@ -654,7 +654,7 @@ class CourtSentencingSynchronisationService(
     nomisSentence.offenderCharges.map {
       mappingApiService.getOffenderChargeByNomisId(it.id).dpsCourtChargeId
     }
-  } catch (notFoundException: WebClientResponseException.NotFound) {
+  } catch (webResponseException: WebClientResponseException) {
```

Comment:
Shouldn't this only be NotFound? a 500 exception I guess shouldn't result in ParentEntityNotFoundRetry you can just allow normal error processing else it might be confusing